core/actions/datacite/action.ts

```diff
@@ -0,0 +1,90 @@
+import * as z from "zod";
+
+import { Action } from "db/public";
+import { Globe } from "ui/icon";
+
+import { defineAction } from "../types";
+
+export const action = defineAction({
+	name: Action.datacite,
+	config: {
+		schema: z.object({
+			doi: z.string().optional(),
+			doiPrefix: z.string().optional(),
+			doiSuffix: z.string().optional(),
+			title: z.string().optional(),
+			url: z.string(),
+			publisher: z.string(),
+			publicationDate: z.date(),
+			creator: z.string(),
+			creatorName: z.string(),
+		}),
+		fieldConfig: {
+			doi: {
+				allowedSchemas: true,
+			},
+			doiSuffix: {
+				allowedSchemas: true,
+			},
+			title: {
+				allowedSchemas: true,
+			},
+			url: {
+				allowedSchemas: true,
+			},
+			publisher: {
+				allowedSchemas: true,
+			},
+			publicationDate: {
+				allowedSchemas: true,
+			},
+			creator: {
+				allowedSchemas: true,
+			},
+			creatorName: {
+				allowedSchemas: true,
+			},
```

Comment:
why do all of these need to be `true`?

Reply:
So you can override their static, configured value with a pub field. Unless I'm misunderstanding something!